components/info_box/ActionBox.tsx

```diff
@@ -51,13 +51,19 @@ const ActionBox = ({
   const [emailError, setEmailError] = useState<EmailErrorCode | null>(null);
   const [checkError, setCheckError] = useState<"MISSING_CONSENT" | null>(null);
   const formRef = useRef<HTMLFormElement | null>(null);
+  const [subCount, setSubCount] = useState<number>(0);
 
   useEffect(() => {
     //string = subscription Date.now()
     //null = not subscribed on client
     //false = on server
     setSubscribed(window.localStorage.getItem("fwd-subscribed"));
-  }, []);
+    fetch(
```

Comment:
I think we might want to add the next.js revalidate config here:
https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating#time-based-revalidation

`fetch('https://...', { next: { revalidate: 3600 } })`

In the top-level layout file I've got `export const revalidate = 0;` to force caching off because Next.js + Storyblok caching weren't playing nicely together. Might be able to change where that's located tbh so it doesn't affect everything globally, but for now any API calls we want cached on the caller side (client -> our server / 3rd party or our server -> 3rd party) probably need to add an explicit `revalidate` option and specify the max cache time.

Reply:
I've added export const revalidate 3600 to this file. 

That seems to work for fetching from DC but doesn't get the browser to cache things. I think this is fine given we have the cache I've created in react (possibly unnecessarily!)

Reply:
I think that's fine here, I don't think we want to cache the users requests? They'll potentially revisit / reload the page and then we want the number to be accurate as soon as possible. 

The main thing is caching the KV lookups since it costs money if we hit it too often.

Reply:
I think we don't want to cache them here since it will just delay how long the user has to wait for the upto date value of the page to load & I don't think it costs us more if people hit the edge cache a lot?